_includes/releases/v22.1/v22.1.14.md

```diff
@@ -0,0 +1,62 @@
+## v22.1.14
+
+Release Date: February 6, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-14-sql-language-changes">SQL language changes</h3>
+
+- [`COPY`](../v22.1/copy-from.html) now logs an error during the insert phase on the [`SQL_EXEC`](../v22.1/logging.html#sql_exec) logging channel. [#95175][#95175]
+- If `copy_from_retries_enabled` is set, [`COPY`](../v22.1/copy-from.html) is now able to retry certain safe circumstances. For example, when `copy_from_atomic_enabled` is false, there is no transaction running `COPY` and the error returned is retriable. [#95505][#95505]
+- `kv.bulkio.write_metadata_sst.enabled` now defaults to false. This change does not affect [`BACKUP`](../v22.1/backup.html) or [`RESTORE`](../v22.1/restore.html). [#96017][#96017]
+
+<h3 id="v22-1-14-db-console-changes">DB Console changes</h3>
+
+- Removed the [**Reset SQL stats**](../v22.1/ui-statements-page.html) and [**Reset index stats**](../v22.1/ui-databases-page.html#index-details) buttons from the DB Console for non-admin users. [#95325][#95325]
+- [Graphs](../v22.1/ui-overview-dashboard.html) can now be clicked on to toggle legend "stickiness" and make the points stop following the mouse. This makes it easier to read dense graphs with many series plotted together. [#94786][#94786]
+
+<h3 id="v22-1-14-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where, in a cluster with nodes running both [v22.2](v22.2.html) and v22.1, range replica changes could sometimes fail on v22.1 leaseholders with the error `change replicas of r47 failed: descriptor changed: [expected] != [actual]`, without any apparent differences between the listed descriptors. Continuing to upgrade all nodes to v22.2 or rolling all nodes back to v22.1 would resolve this issue. [#94841][#94841]
```

Comment:
'range replica' could be a link to https://www.cockroachlabs.com/docs/stable/ui-replication-dashboard.html#review-of-cockroachdb-terminology